web_widget_x2many_2d_matrix/static/src/js/web_widget_x2many_2d_matrix.js

```diff
@@ -50,10 +53,18 @@ openerp.web_widget_x2many_2d_matrix = function(instance)
             this.field_y_axis = node.attrs.field_y_axis || this.field_y_axis;
             this.field_label_x_axis = node.attrs.field_label_x_axis || this.field_x_axis;
             this.field_label_y_axis = node.attrs.field_label_y_axis || this.field_y_axis;
+            this.x_axis_clickable = node.attrs.x_axis_clickable != undefined ? this.parse_boolean(node.attrs.x_axis_clickable) : this.x_axis_clickable;
+            this.y_axis_clickable = node.attrs.y_axis_clickable != undefined ? this.parse_boolean(node.attrs.y_axis_clickable) : this.y_axis_clickable;
```

Comment:
Like in Python, `"0"` is true. Not sure if it applies here, otherwise ignore this :grin: